libs/solid/profile/src/lib/services/intro.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Inject, Injectable } from '@angular/core';
```

Comment:
das heißt: in jeder Komponente, in der wir eine Tour haben wollen, muss es einen `intro.service` geben?

Ich vermute, dass wir das vereinheitlichen könnten, so dass der Service nur an einer Stelle steht. 
Aber ich glaube, das hattest Du vorher und es gibt einen Grund, warum es jetzt anders ist.

ABER - viel wichtiger: Make it work first and refactor later. 
Also nicht zu viel Zeit mit Kosmetik verbringen, sondern erst Funktionalität programmieren (auch, wenn es nicht super schön ist).

Reply:
...auch, wenn ich hier Vorschläge für Variablennamen mache 😉